lesson04/assignment/css/_main.content.scss

```diff
@@ -0,0 +1,321 @@
+* {
+    box-sizing: border-box;
+    @include mrgn-pdng-set-0;
+    position: relative;
+}
+
+body {
+    @include full-x-y;
+    color: $white;
+}
+
+.Main {
+    // header layout
+    .header {

```

Comment:
Your SCSS is very clean, good use of nesting

Reply:
Thanks Tyler. This time, I tried to polish the code before pushing into the lesson's branch.